client_with_ssl.go

```diff
@@ -5,24 +5,20 @@ import (
 	"crypto/tls"
 	"fmt"
 	"github.com/hazelcast/hazelcast-go-client"
+	"github.com/hazelcast/hazelcast-go-client/serialization"
+	"github.com/hazelcast/hazelcast-go-client/sql"
 	"github.com/hazelcast/hazelcast-go-client/types"
 	"log"
 	"math/rand"
-	"os"
 	"path/filepath"
 	"time"
 )
 
-/**
-*
-* This is boilerplate application that configures client to connect Hazelcast Cloud cluster.
-* After successful connection, it puts random entries into the map.
-*
-* See: https://docs.hazelcast.cloud/docs/go-client
-*
- */
+// This is boilerplate application that configures client to connect Hazelcast Cloud cluster.
+// After successful connection, it runs the uncommented examples.
+//
+// See: https://docs.hazelcast.cloud/docs/go-client
 func main() {
-	_ = os.Setenv("HZ_CLOUD_COORDINATOR_BASE_URL", "YOUR_DISCOVERY_URL")
```

Comment:
:eye: https://github.com/hazelcast/hazelcast-cloud-go-sample-client/pull/12
This (as you probably know) is used by non-production environments, for setting up API url

Reply:
Right, refactored at https://github.com/hazelcast/hazelcast-cloud-go-sample-client/pull/13/commits/c5ce03ef900c86e0dcbfd47b4910e698a6dd5370.